Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/SubmissionResults.tsx

```diff
@@ -42,24 +44,33 @@ const SubmissionResults = ({ problemId }: ISubmissionResultsProps) => {
         }
     }, [ getSubmissionResults, problemId ]);
 
-    useEffect(() => {
+    const reload = () => {
         (async () => {
             await getResults();
         })();
+    };
```

Comment:
No need for the IIFE here.
Also, use `useCallback`

Reply:
oops, copy paste mistake